server.js

```diff
@@ -10,11 +11,65 @@ app.use(cors());
 const bookings = require("./bookings.json");
 
 app.get("/", function (request, response) {
-  response.send("Hotel booking server.  Ask for /bookings, etc.");
+response.send("Hotel booking server.  Ask for /bookings, etc.");
+});
+// gets all bookings 
+app.get("/bookings", (req, res) => {
+res.send(bookings);
+});
+
+app.post("/bookings", (req, res) => {
+const { title, firstName, surname, email, checkInDate, checkOutDate} = req.body;
+if (!title || !surname || !email || !checkInDate || !checkOutDate){
+res.status(400).send({message: " ERROR All fields have not been completed please try again"});
+}
+let newBookingTable = {
+id: 0,
+title: req.body.title,
+firstName: req.body.firstName,
+surname: req.body.surname,
+email: req.body.email,
+roomId: 0,
+checkInDate: req.body.checkInDate,
+checkOutDate: req.body.checkOutDate,
+}
+bookings.push(newBookingTable);
+newBookingTable.id = bookings.findIndex(newBooking) + 1;
+newBookingTable.roomId = newBookingTable.id + 10;
+res.send(newBookingTable);
+res.json(bookings);
```

Comment:
This will send 2 responses for one request (the first with `res.send`, the second with `res.json`) and the server will crash.